XpringKit/PayID/PayIDClient.swift

```diff
@@ -0,0 +1,80 @@
+import Alamofire
+import Foundation
+
+/// Implements interaction with a PayID service.
+/// - Warning:  This class is experimental and should not be used in production applications.
+public class PayIDClient {
+  /// The network this PayID client resolves on.
+  private let network: XRPLNetwork
```

Comment:
So this is the non-payment-network agnostic implementation, right?

Reply:
For now yes.